telluric/georaster.py

```diff
@@ -1326,14 +1326,27 @@ def _resize(self, ratio_x, ratio_y, resampling):
         """Return raster resized by ratio."""
         new_width = int(np.ceil(self.width * ratio_x))
         new_height = int(np.ceil(self.height * ratio_y))
-
         dest_affine = self.affine * Affine.scale(1 / ratio_x, 1 / ratio_y)
-        if self.not_loaded():
+
+        raster = self._raster_backed_by_a_file()
+
+        with self._raster_opener(raster._filename) as src:
             window = rasterio.windows.Window(0, 0, self.width, self.height)
-            resized_raster = self.get_window(window, xsize=new_width, ysize=new_height, resampling=resampling)
-        else:
-            resized_raster = self._reproject(new_width, new_height, dest_affine, resampling=resampling)
```

Comment:
It seems that with the new code, the `resampling` parameter is actually ignored
I wonder:
(a) What resampling in actually does when the scaling is not integer? Can that be controlled?
(b) If it cannot be controlled, then at least we should check if the requested sampling is equal to the one actually used (if not - should either fail, or revert to the slower method)

Reply:
Nice catch! I've brought it back. But since we get rid of reprojection resampling options `max`, `min`, `med`, `q1` and `q3` are not available anymore for `GeoRaster2.resize`. If you need them you should use `GeoRaster2.reproject`.